docs/docs/development/documentation.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Documentation
+sidebar_label: Documentation
+---
+
+This document outlines how to test your documentation changes locally and prepare the changes for a pull request.
+
+The documentation is written with [Docusaurus](https://docusaurus.io/). The ZMK source code has all of the necessary Docusaurus dependencies included but referencing their documentation can be helpful at times.
+
+The general process for updating the ZMK documentation is:
+
+1. Update the documentation
+2. Test the changes locally
+3. Ensure the sources are formatted properly and linted
+4. Create a Pull Request for review and inclusion into the ZMK sources
+
+:::note
+If you are working with the documentation from within VS Code+Docker please be aware the documentation will not be auto-generated when making changes while the server is running. You'll need to restart the server when saving changes to the documentation.
+:::
+
+## Preparation
+
+After making updates to the documentation locally, you'll want to test the updates locally using the following procedure. The `npm` commands and first step will need to be run from a terminal.
+
+## Testing Documentation Updates Locally
+
+To verify documentation updates locally, follow the following procedure. The `npm` commands and first step will need to be run from a terminal.
```

Comment:
This seems repetitive. I see that the first section is supposed to outline the updating of documentation and the next is testing the changes, but I'm wondering if the "Preparation" section can be removed or maybe could be changed to a small write up about editing files?

Reply:
This is definitely redundant now that npm is included in the dev container and elsewhere. I'm going to go ahead and remove the preparation section in favor of an "updating" section that includes some of the early on notes.